domain/src/main/java/org/oppia/domain/analytics/AnalyticsController.kt

```diff
@@ -0,0 +1,143 @@
+package org.oppia.domain.analytics
+
+import android.content.Context
+import org.oppia.app.model.EventLog
+import org.oppia.app.model.EventLog.EventAction
+import org.oppia.app.model.EventLog.Priority
+import org.oppia.util.logging.EventLogger
+import javax.inject.Inject
+
+const val TEST_TIMESTAMP = 1556094120000
+const val TEST_TOPIC_ID = "test_topicId"
+const val TEST_STORY_ID = "test_storyId"
+const val TEST_EXPLORATION_ID = "test_explorationId"
+const val TEST_QUESTION_ID = "test_questionId"
```

Comment:
Why are these constants here? We shouldn't keep test ids in the main file, they should be in the test file

Reply:
Okay, got it. Done.